spec/unit/puppet_x/chocolatey/chocolatey_common_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+require 'puppet_x/chocolatey/chocolatey_install'
+
+describe 'Chocolatey Common' do
+
+  context 'on Windows', :if => Puppet::Util::Platform.windows? do
+
```

Comment:
This is empty right now, but I wanted to get eyes on this so we can get it merged and through soon. So we can pick this up in other tickets or as maint.